reckoner/chart.py

```diff
@@ -229,7 +230,6 @@ def install(self, namespace=None, context=None):
             logging.info(r.stdout)
         except HelmClientException, e:
             logging.error(e)
-            raise e
```

Comment:
Did you remove the re-raise on purpose?

Reply:
Yes, it seemed somewhat unnecessary.  If it needs to be there please let me know

Reply:
I _think_ the initial intent of re-raising was so you could log the error and then handle it gracefully at "top": https://github.com/reactiveops/reckoner/blob/master/reckoner/course.py#L119 but it doesn't look like that ever got finished. 

However, as it stands now, the error will be logged and then the post install hook on https://github.com/reactiveops/reckoner/blob/master/reckoner/course.py#234 will run even though the `Helm.Client().upgrade` threw a `HelmClientException`. That is probably not the intended behavior. 

Adding a `return` or `raise` will stop the post_install_hook from running.

Reply:
This sounds like a test case. #allHooksRunEvenIfInstallFails